test/protobuf/protoc/integration_test.exs

```diff
@@ -55,4 +55,12 @@ defmodule Protobuf.Protoc.IntegrationTest do
   test "extensions" do
     assert "hello" == Protobuf.Protoc.ExtTest.Foo.new(a: "hello").a
   end
+
+  describe "message extensions" do
+    test "uses the option type_name" do
+      file = File.read!("test/protobuf/protoc/proto_gen/extension.pb.ex")
+
+      assert file =~ "inserted_at: DateTime.t() | nil\n"
```

Comment:
Very weak test, but I don't how is the best way to test generation from protobuf file here.